src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1333,16 +1333,11 @@ private RendezvousAwaitable<int> InternalSendRequestAsync(WinHttpRequestState st
                     0,
                     state.ToIntPtr()))
                 {
-                    int lastError = Marshal.GetLastWin32Error();
```

Comment:
I am removing this one and other location that I see was added during the time of the investigation of the original bug, let me know if there is a reason to keep them.